src/Model/Contact.php

```diff
@@ -1700,6 +1700,10 @@ private static function checkAvatarCacheByArray(array $contact, bool $no_update
 			}
 		}
 
+		if (Network::isLocalLink($contact['url']) && !empty($contact['avatar'])) {
```

Comment:
The "url" field is not defined on my system for the `$contact` causing at least a PHP warning

```
PHP Warning:  Undefined array key "url" in src/Model/Contact.php on line 1703
```

When calling a DM conversation that was started before tonight's pull from the RC branch.

Reply:
The friendica log has the following along side the above PHP error

```
Uncaught Exception TypeError: "Friendica\Util\Network::isLocalLink(): Argument #1 ($url) must be of type string, null given, called in /src/Model/Contact.php on line 1703" at /src/Util/Network.php line 559 
{"exception":"TypeError: Friendica\\Util\\Network::isLocalLink(): Argument #1 ($url) must be of type string, null given, called in src/Model/Contact.php on line 1703 and defined in /src/Util/Network.php:559\nStack trace:

0 /src/Model/Contact.php(1703): Friendica\\Util\\Network::isLocalLink()
1 /src/Model/Contact.php(1607): Friendica\\Model\\Contact::checkAvatarCacheByArray()
2 /src/Model/Contact.php(1653): Friendica\\Model\\Contact::getAvatarPath()
3 /mod/message.php(312): Friendica\\Model\\Contact::getThumb()
4 /src/LegacyModule.php(96): message_content()
5 /src/LegacyModule.php(73): Friendica\\LegacyModule->runModuleFunction()
6 /src/BaseModule.php(239): Friendica\\LegacyModule->content()
7 /src/App.php(711): Friendica\\BaseModule->run()\n#8 /index.php(44): Friendica\\App->runFrontend()
9 {main}"} - {"file":null,"line":null,"function":null,"uid":"1b3bd7","process_id":7457}
```